js/main.js

```diff
@@ -1,44 +1,47 @@
-let matsList = ''; // final matList output
-let elementAmount = {}; // user input storeage
-
 function copyMatsList() { // copies matsList to clipboard
-    collectAmount();
-    calculateTotalMats();
-    toList();
+    let elementAmount = collectAmount();
+    let collection = calculateTotalMats(elementAmount);
+    let matsList = toList(collection);
     let copyList = document.createElement("textarea"); // creates textarea element
     document.body.appendChild(copyList); // attaches textarea element to HTML body
     copyList.value = matsList;
     copyList.select();
     document.execCommand("copy"); // copies to clipboard
     document.body.removeChild(copyList); // removes textarea from HTML body
-    reset();
 }
 
-function toList() { // converts collection Object to string matsList
-    for (let key of Object.keys(collection)) { // key = each integer in collection Object
-        if (collection[key] === 0) { // if key is 0 contiune (skips to next key)
-            continue;
-        }
-        matsList += `${collection[key]}x ${key} \n`; // sets matsList formating
+function toList(collection) { // converts collection Object to string matsList
+    let matsList = '';
+    let displayList = '';
+    for (let key of Object.keys(collection)) { // key = each key in collection Object

```

Comment:
Wait until you learn about using `Object.entries()`... 😄

Reply:
oh no